components/x-timeline-feed/src/TimelineFeed.scss

```diff
@@ -0,0 +1,51 @@
+@import 'o-colors/main';
+@import 'o-grid/main';
+@import 'o-typography/main';
+
+.articleGroup {
+	border-top: 4px solid #000;
+
+	@include oGridRespondTo($from: M) {
+		display: grid;
+		grid-gap: 0 20px;
+		grid-template: "heading articles" min-content / 1fr 3fr;
+	}
+}
+
+.articleGroup__heading {
+	@include oTypographySansBold($scale: 1);
+	@include oTypographyMargin($top: 2, $bottom: 2);
+
+	@include oGridRespondTo($from: M) {
+		grid-area: heading;
```

Comment:
Minor thing, but I wonder if the grid template itself should change at the various viewports, rather than the elements within opting in & out of it?

Reply:
I think it's better to not use CSS grid unless its necessary. I don't feel strongly about it though.